infra/nnfw/cmake/options/options_aarch64-android.cmake

```
@@ -1,4 +1,7 @@
 # aarch64 android cmake options
 #
+option(BUILD_ARMCOMPUTE "Build ARM Compute from the downloaded source" ON)
+option(DOWNLOAD_ARMCOMPUTE "Build ARM Compute from the downloaded source" ON)
```

It is `DOWNLOAD_ARMCOMPUTE`, but the description seems to be `Build ...`. It would be better to change to `Download..`
```suggestion
option(DOWNLOAD_ARMCOMPUTE "Download ARM Compute source" ON)
```